src/test/java/io/reactivex/single/SingleTest.java

```diff
@@ -22,6 +22,7 @@
 import org.junit.*;
 
 import io.reactivex.*;
+import io.reactivex.Observable;
```

Comment:
needed to add this import otherwise:

`SingleTest.java:562: error: reference to Observable is ambiguous Single.fromObservable(Observable.just(1))`